Sources/NIOHTTP2/HTTP2CommonInboundStreamMultiplexer.swift

```diff
@@ -109,8 +109,8 @@ extension HTTP2CommonInboundStreamMultiplexer {
             // does no actual work.
             self.streamChannelContinuation?.yield(channel: channel.baseChannel)
 
-            channel.configureInboundStream(initializer: self.inboundStreamStateInitializer)
             channel.receiveInboundFrame(frame)
+            channel.configureInboundStream(initializer: self.inboundStreamStateInitializer)
```

Comment:
This is quite subtle. Can we add a comment here that this order is important.

Reply:
Absolutely; done